.travis.yml

```diff
@@ -46,6 +46,12 @@ script:
     time BATS_NO_PARALLELIZE_ACROSS_FILES=1 BATS_NUMBER_OF_PARALLEL_JOBS=2 bin/bats --tap test
   fi
 
+  if [[ "${TRAVIS_OS_NAME:-}" == 'linux' && "${BASHVER:-}" == 5 ]]; then
```

Comment:
Did you intentionally pull in @sublimino's commit? The PR title does not suggest so.